Library/Homebrew/formula_cellar_checks.rb

```diff
@@ -340,17 +342,17 @@ def check_binary_arches(formula)
 
     if mismatches.present?
       s += <<~EOS
-        Binaries built for an incompatible architecture were installed into #{formula}'s prefix.
+        Binaries built for a non-native architecture were installed into #{formula}'s prefix.
         The offending files are:
-          #{mismatches * "\n  "}
+          #{mismatches.map { |m| "#{m.first}\t(#{m.last})" } * "\n  "}
```

Comment:
The `\t` is a crude attempt at getting some sort of alignment of the output. It's not perfect, but I think it suits for what we're using this for.

Also, this still shows `(universal)` as the arch for universal binaries that don't contain a native slice. Not sure if we want to show the actual slices, so I've left it this way for now.

Reply:
I think this is fine.

Another option could be to put the type first:

```
foo:
  * Binaries built for a non-native architecture were installed into foo's prefix.
    The offending files are:
      (i386)          /usr/local/Cellar/foo/1.0/lib/libbar.dylib
      (arm64)         /usr/local/Cellar/foo/1.0/lib/libfoo.dylib
      (universal)     /usr/local/Cellar/foo/1.0/lib/libincompatible.dylib
    Unexpected universal binaries were found.
    The offending files are:
      /usr/local/Cellar/foo/1.0/lib/liball.dylib
      /usr/local/Cellar/foo/1.0/lib/libuniversal.dylib
```

I'm not sure I like this any better, though. Probably just a personal preference thing

Reply:
Yea, I'm fine with slightly misaligned output when some `Pathname`s are too long. It's still readable anyway, which is what I was going for with alignment (compare this with the case where I only leave a single space as in `"#{m.first} (#{m.last})"`).